src/domain/event/CreateEventPage.tsx

```diff
@@ -128,49 +157,104 @@ const CreateEventPage: React.FC = () => {
             toast.error(t('errors.serverError'));
         }
       }
-      // eslint-disable-next-line no-console
-      console.error('Failed to save event with error', e);
     }
   };
 
   return (
     <Formik
       initialValues={EVENT_INITIAL_VALUES}
-      onSubmit={(values) => {
-        saveEvent(values, PublicationStatus.Public);
-      }}
-      validationSchema={createEventValidationSchema}
+      // We have custom way to handle onSubmit so here is empty function
+      // to silent TypeScript error. The reason for custom onSubmit is that
+      // we want to scroll to first invalid field if error occurs
+      onSubmit={/* istanbul ignore next */ () => undefined}
+      validationSchema={eventValidationSchema}
       validateOnMount
+      validateOnBlur={true}
+      validateOnChange={true}
     >
-      {({ values: { type, ...restValues }, setErrors }) => {
+      {({ values: { type, ...restValues }, setErrors, setTouched }) => {
+        const getFocusableFieldId = (fieldName: string): string => {
+          // For the select elements, focus the toggle button
+          if (
+            fieldName === EVENT_FIELDS.KEYWORDS ||
+            fieldName === EVENT_FIELDS.LOCATION ||
+            fieldName === EVENT_FIELDS.SUPER_EVENT
```

Comment:
maybe create select elements array and use .find function here

Reply:
Nice idea. Added an array and used it here to find is the fieldName in the list or not